native_client/javascript/client.ts

```diff
@@ -78,23 +72,24 @@ const buffer = Fs.readFileSync(args['audio']);
 const result = Wav.decode(buffer);
 
 if (result.sampleRate < desired_sample_rate) {
-  console.error('Warning: original sample rate (' + result.sampleRate + ') ' +
-                'is lower than ' + desired_sample_rate + 'Hz. ' +
-                'Up-sampling might produce erratic speech recognition.');
+  console.error(`Warning: original sample rate ( ${result.sampleRate})` +
+                `is lower than ${desired_sample_rate} Hz. ` +
+                `Up-sampling might produce erratic speech recognition.`);
 }
 
-function bufferToStream(buffer) {
+function bufferToStream(buffer: Buffer) {
   var stream = new Duplex();
   stream.push(buffer);
   stream.push(null);
   return stream;
 }
 
-var audioStream = new MemoryStream();
+let audioStream = new MemoryStream();
 bufferToStream(buffer).
   pipe(Sox({
     global: {
       'no-dither': true,
+      'replay-gain': 'off',
```

Comment:
FYI: https://github.com/ArtskydJ/sox-stream/pull/11#discussion_r402506159

Reply:
This was flagged as optional by `tsc`, guess I'll make sure we update the dep and remote that.

Reply:
Hm, no new release.